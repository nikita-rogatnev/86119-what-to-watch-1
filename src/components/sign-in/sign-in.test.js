import React from 'react';
import renderer from 'react-test-renderer';

import SignIn from './sign-in';

it(`Sign In renders correctly`, () => {
  const tree = renderer.create(<SignIn/>).toJSON();

  expect(tree).toMatchSnapshot();
});
