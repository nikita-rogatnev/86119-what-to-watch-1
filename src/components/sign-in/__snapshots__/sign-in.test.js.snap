// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn renders correctly 1`] = `
<section
  className="sign-in user-page__content"
>
  <form
    className="sign-in__form"
    onSubmit={[Function]}
  >
    <div
      className="sign-in__fields"
    >
      <div
        className="sign-in__field "
      >
        <input
          autoComplete="username email"
          className="sign-in__input"
          id="user-email"
          name="user-email"
          onKeyUp={[Function]}
          placeholder="Email address"
          required="required"
          type="email"
        />
        <label
          className="sign-in__label visually-hidden"
          htmlFor="user-email"
        >
          Email address
        </label>
      </div>
      <div
        className="sign-in__field"
      >
        <input
          autoComplete="current-password"
          className="sign-in__input"
          id="user-password"
          name="user-password"
          onKeyUp={[Function]}
          placeholder="Password"
          required="required"
          type="password"
        />
        <label
          className="sign-in__label visually-hidden"
          htmlFor="user-password"
        >
          Password
        </label>
      </div>
    </div>
    <div
      className="sign-in__submit"
    >
      <button
        className="sign-in__btn"
        type="submit"
      >
        Sign in
      </button>
    </div>
  </form>
</section>
`;
